README.md

```diff
@@ -61,10 +61,10 @@ Checklist of the most important security countermeasures when designing, testing
 - [ ] Design a rollback solution for deployments.
 
 ## Monitoring
-- [ ] Use centralized loggins for all services and components.
+- [ ] Use centralized log-ins for all services and components.
 - [ ] Use agents to monitoring all trafic, errors, requests and responses.
 - [ ] Use alerts for SMS, Slack, Email, Telegram, Kibana, Cloudwatch e etc.
```

Comment:
Should read:

> Use alerts for SMS, Slack, Email, Telegram, Kibana, Cloudwatch, etc.

Extra `e` is not required and as there is no conjunction after `etc` it should have a comma preceding it.

👍